locomotion-sdk/src/pages/AuthScreens/Onboarding/Card.js

```diff
@@ -8,16 +8,14 @@ import onboardingContext from '../../../context/onboarding';
 import PaymentsContext from '../../../context/payments';
 
 const Card = () => {
-  const { navigateBasedOnUser, onboardingState } = onboardingContext.useContainer();
-  const { getPaymentMethods } = PaymentsContext.useContainer();
+  const { nextScreen, onboardingState } = onboardingContext.useContainer();
```

Comment:
we have to kill this context or rename it to user context, 
we have to use `route.params` so we can re use this pages else where (in `Account` page @avivbarniv fyi)

Reply:
why is onboarding context bad? we can reuse the page components for different screens no?

Reply:
to make it reusable we need to pass the next page as a param for example